lib/DetectRPI.js

```diff
@@ -23,7 +23,8 @@ module.exports = function() {
     "0010" : "RPIB+",
     "0011" : "CM",
     "a11041" : "RPI2B",
-    "a21041" : "RPI2B"
+    "a21041" : "RPI2B",
+    "a02082" : "RPI3B"
```

Comment:
@snazzybunny we should also add comments for raspberry pi zero and pi zero W, so that other maybe fill this gaps ...